WordPress/src/main/java/org/wordpress/android/ui/accounts/AbstractFragment.java

```diff
@@ -301,6 +309,10 @@ public void onErrorResponse(VolleyError error) {
             } else {
                 showError(messageId);
             }
+
+            if (mStep == Step.CREATE_USER) {
+                AnalyticsTracker.track(AnalyticsTracker.Stat.CREATE_ACCOUNT_FAILED);
```

Comment:
What's about tracking the network error details in properties?
Something like the following code (that I didn't test)
```
Map<String, Object> properties = new HashMap<>();
properties.put("network_exception_class", error != null ? error.getClass().getCanonicalName() : "null");
properties.put("network_exception_message", error != null ?  error.getMessage() : "null");
if (error.networkResponse != null) {
     NetworkResponse networkResponse = error.networkResponse;
     if (networkResponse.data != null) {
             properties.put("network_exception_data",  new String(networkResponse.data)););
      }
}
```